### ISSUE TITLE
clean shutdown

### ISSUE THREAD

**Issue by edkangdev** [JaneliaAPIG/spoke#9]
make shutdown clean, right now user has to close all, clear all, clc

**Comment by JaneliaAPIG** [comment]
When you don't do a clean shutdown you get an error stack on restarting Spoke, which begins with:

```
Error using most.gui.AdvancedPanelToggler.init (line 28)
Unexpected figure userdata.
```

**Comment by JaneliaAPIG** [comment]
Note that the following works cleanly to shutdown Spoke & allow clean restart
`delete(hGrid)`

**Comment by vijayiyer05** [comment]
Commit e4333ae

**Comment by vijayiyer05** [comment]
Fixed by b3515d658d922556862d17af6729a96e3ece7eff
